electron-app/src/server/websites/hentai-foundry/hentai-foundry.service.ts

```diff
@@ -59,7 +59,11 @@ export class HentaiFoundry extends Website {
   }
 
   getScalingOptions(file: FileRecord): ScalingOptions {
-    return { maxSize: FileSize.MBtoBytes(50) };
+    return { 
+      maxHeight: 1500,
+      maxWidth: 1500,
+      maxSize: FileSize.MBtoBytes(2)
```

Comment:
Is the limit reduced now?

Reply:
Did some digging after it was mentioned on discord and spoke to mods who confirmed lower limits. Also mentioned on the forums: https://forums.hentai-foundry.com/viewtopic.php?t=2681; anything above this is basically limited approval only.